sql/ddl/alter/sql-practice-define-default-seq-4.md

```diff
@@ -0,0 +1,35 @@
+---
+author: SebaRaba
+
+levels:
+  - beginner
+  - basic
+  - medium
+
+tags:
+  - introduction
+  - workout
+  - deep
+
+type: exercise
+
+linkType: sqlfiddle
+
+link: http://sqlfiddle.com/#!9/a3e1d
+
+standards:
+  sql.define-sequences-or-default-values.1: 1000
+  sql.define-sequences-or-default-values.2: 1000
+
+links:
+  - '[mysql modify columns documentation](https://dev.mysql.com/doc/refman/5.7/en/create-index.html){website}'
+  - '[blog post on mysql define tables](https://atech.blog/viaduct/mysql-indexes-primer){website}'
+  - '[video tutorial on aggregate functions](https://www.youtube.com/watch?v=JjNef3tPltU){video}'
+
+---
+# SQL Practice Alter Table
+---        
```

Comment:
seems like it doesn't fail, but nem instructed us to put a `\n` above sections `---`